src/Sulu/Bundle/AudienceTargetingBundle/Entity/TargetGroup.php

```diff
@@ -0,0 +1,224 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AudienceTargetingBundle\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+
+/**
+ * Entity class for target group.
+ */
+class TargetGroup implements TargetGroupInterface
+{
+    /**
+     * @var int
+     */
+    private $id;
+
+    /**
+     * @var string
+     */
+    private $title;
+
+    /**
+     * @var string
+     */
+    private $description;
+
+    /**
+     * @var int
+     */
+    private $priority;
+
+    /**
+     * @var bool
+     */
+    private $allWebspaces = false;
+
+    /**
+     * @var bool
+     */
+    private $active = false;
+
+    /**
+     * @var Collection
+     */
+    private $webspaces;
+
+    /**
+     * @var Collection
+     */
+    private $rules;
```

Comment:
Yes, it's not implemented in this PR yet. But i don't see why i shouldn't create the full entity right from the beginning, when i'm implementing the db schema.